src/main/java/com/reshiftsecurity/plugins/intellij/gui/toolwindow/view/ToolWindowPanel.java

```diff
@@ -163,7 +163,7 @@ private void updateLayout(final boolean enablePreviewLayout) {
 		if (!_isPreviewLayoutEnabled && enablePreviewLayout) {
 			_updateMultiSplitLayout(PREVIEW_LAYOUT_DEF);
 			getMultiSplitPane().add(getBugTreePanel(), "left");
-			_multiSplitPane.add(getPreviewPanel().getComponent(), "middle");
+			getMultiSplitPane().add(getPreviewPanel().getComponent(), "middle");
```

Comment:
Suggestion, minor: Don't call the getter every time, store the result somewhere?

Reply:
For this one the getter just ensures the private local variable is not null. I just made it consistent throughout the file.